### PR TITLE
Support maxTries if define in queue jobs. 

### PR DESCRIPTION
**Pull request by gtlkirit**

### CODE REVIEW

**Review by eneskaya — APPROVED** [review]

File: src/Controllers/WorkerController.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ public function queue(Request $request, WorkerInterface $worker, Container $lara
 
         $worker->process(
             $request->header('X-Aws-Sqsd-Queue'), $job, [
-                'maxTries' => 0,
+                'maxTries' => (isset($body['maxTries']) && !is_null($body['maxTries']))? $body['maxTries'] : 0,
```
Comment (dusterio) [inline src/Controllers/WorkerController.php]: hmm, but isn't $body a string?

Comment (gtlkirit) [reply src/Controllers/WorkerController.php]: Yes, true, its return string not array. need to change.

Comment (gtlkirit) [reply src/Controllers/WorkerController.php]: @dusterio Fixed it.

File: src/Controllers/WorkerController.php [hunk 2]
```diff
@@ -77,9 +77,11 @@ public function queue(Request $request, WorkerInterface $worker, Container $lara
             ]
         ]);
 
+        $bodyArray = json_decode($body, true);
+
         $worker->process(
             $request->header('X-Aws-Sqsd-Queue'), $job, [
-                'maxTries' => 0,
+                'maxTries' => (isset($bodyArray['maxTries']) && !is_null($bodyArray['maxTries']))? $bodyArray['maxTries'] : 0,
```
Comment (fylzero) [inline src/Controllers/WorkerController.php:84]: Can use a simple null coalescing operator here:

```suggestion
                'maxTries' => $bodyArray['maxTries'] ?? 0,
```

Comment (fylzero) [inline src/Controllers/WorkerController.php:84]: !is_null is redundant here and should be removed.

**Comment by fylzero** [comment]
Marking this PR as stale.

**Pull request closed without merging**